### PR TITLE
sql password special characters handled

### PR DESCRIPTION
**Pull request by rokmicrosoft**

### CODE REVIEW

**Comment by rokmicrosoft** [comment]
To be consistent with[ sql deployment task](https://github.com/Microsoft/vsts-rm-extensions/blob/master/TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1#L59) Used same logic. Tested with username (`Ros'2016) and password(`$Ro`s"'2016) working fine.

File: Tasks/SqlAzureDacpacDeploymentV1/DeploySqlAzure.ps1 [hunk 1]
```diff
@@ -58,6 +58,15 @@ function ThrowIfMultipleFilesOrNoFilePresent($files, $pattern)
     }
 }
 
+function EscapeSpecialChars
+{
+    param(
+        [string]$str
+    )
+
+    return $str.Replace('`', '``').Replace('$', '`$')
```
Comment (Ajay-MS) [inline Tasks/SqlAzureDacpacDeploymentV1/DeploySqlAzure.ps1]: Can you verify once for single quotes too in the password?

Comment (rokmicrosoft) [reply Tasks/SqlAzureDacpacDeploymentV1/DeploySqlAzure.ps1]: Already verified $Ro`s"'2016 .It also contains single quote .

Comment (Ajay-MS) [reply Tasks/SqlAzureDacpacDeploymentV1/DeploySqlAzure.ps1]: :)

**Comment by Ajay-MS** [comment]
L0 test failing - SqlAzureDacpacDeployment  Suite (SQL) Should run successfully for all valid inputs.

**Pull request merged**